build/linux/Dockerfile

```diff
@@ -8,4 +8,9 @@ WORKDIR /
 
 EXPOSE 9000
 
-ENTRYPOINT ["/portainer"]
+ENTRYPOINT ["./portainer"]
```

Comment:
It allows to extract portainer anywhere in the image/container and have it started according to WORKDIR. I think that, if this change is not applied, WORKDIR should be removed from the Dockerfile, because it is not required/used at all.

Reply:
I believe this is out of scope of the PR, please report a new issue to discuss it first.